src/sentry/static/sentry/app/views/issueList/overview.jsx

```diff
@@ -220,7 +218,7 @@ const IssueListOverview = createReactClass({
 
   getGroupStatsPeriod() {
     const currentPeriod = this.props.location.query.groupStatsPeriod;
-    return STATS_PERIODS.has(currentPeriod) ? currentPeriod : DEFAULT_GRAPH_STATS_PERIOD;
+    return STATS_PERIODS.has(currentPeriod) ? currentPeriod : DEFAULT_STATS_PERIOD;
```

Comment:
This looks like it would change for all users not just those that are exposed to the new issue counts. Is that ok?

Reply:
I'll revert these changes, I have another pr I'll put up that has more substantial changes to these